WooCommerce/Classes/Authentication/Epilogue/StorePickerCoordinator.swift

```diff
@@ -139,6 +163,7 @@ private extension StorePickerCoordinator {
 
     /// Switches the current user's default store to the one having the provided `storeID`.
     /// After successfully switching, the store picker screen should be dismissed.
+    @available(*, renamed: "switchStore(with:)")
```

Comment:
just curious, what does this renaming syntax do?

Reply:
Xcode added this when I used it to add an async wrapper. Removed it in fa2207c8af1f8a3ed0746d5abac2252e3c3b6400